de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/InventoryAllocationDestination.java

```diff
@@ -372,14 +384,55 @@ private I_M_InventoryLine getCreateInventoryLine(final I_M_InOutLine inOutLine,
 		return inventoryLine;
 	}
 
-	private BigDecimal countTUs(final IHUContext huContext, final I_M_HU hu, final I_M_InOutLine receiptLine)
+	private BigDecimal countTUs(final IHUContext huContext, final I_M_HU hu, final I_M_InventoryLine inventoryLine)
 	{
+		// Find out if the hu is VHU or Aggregate -> Move to TU level
+		// Add the TU recursively
```

Comment:
i don't understand this comment...assume the `hu` is virtual: 
i understand that in this case its parent is the TU that you actually need. 
But what does "aggregate" mean in this context?